File: fotocopiadora.cs
using System;
/////////////////////
class fotocopiadora
{

    static void main()
    {
        int hojas;
        double total;


    
////////////////////////
        Console.clear();
////////////////////////
        Console.write("ingresa la cantidad de hojas por favor");
        hojas - int.Parse(Console.ReadLine());
////////////////////////
        total = 20 * hojas;
        if (hojas > 30)
        {
            total*= 0.90;
        }
//////////////////////////
        Console.writeLine("Presiona una tecla para continuar")
        Console.ReadKey();
//////////////////////////
        Console.Clear();
//////////////////////////                
        Console.writeLine("\n *****BOLETA*******");
        Console.writeLine("$ cantidad de hojas {hojas}");
        Console.writeLine("valor por hojas $20");
        Console.writeLine("TOTAL $ {total: F2}");
    }
}
/////////////////////////

File: fotocopiadora.R
clear_screen <- function() {
    if (.Platform$OS.type == "unix") {
        system("clear")
    } else {
        system("cls")
    }
}
############################################
clear_screen()
#############################################
hojas <- as.integer(readline(prompt = "ingresa el numero de hojas: "))
#############################################
precio_por_hoja <- 20
total <- precio_por_hoja*hojas
#############################################
uf (hojas>30) {
    total <- total*0.90
}
#############################################
cat("\npresiona enter para continuar")
readline()
#############################################
clear_screen()
##############################################
cat("\n ********boleta********")
cat(" cantidad de hojas: ", hojas, "\n")
cat("precio por hoja:  $20\n")
cat("total a pagar: $", total, "\n")
##############################################
